gravitee-apim-console-webui/src/management/api/analytics/logs/log.component.ts

```diff
@@ -208,8 +208,12 @@ class LogComponentController {
     const queryParamsMap = uri
       .slice(uri.indexOf('?') + 1)
       .split('&')
-      .map((queryParamsAsString) => queryParamsAsString.split('='))
-      // Convert in a map to group query params with the same key (it can happens when sending an array), for the example:
+      .map((queryParamsAsString) => {
+        // A simple `.split` is not enough as query param values can contains `=` themselves
+        const indexOfEqualChar = queryParamsAsString.indexOf('=');
+        return [queryParamsAsString.substring(0, indexOfEqualChar), queryParamsAsString.substring(indexOfEqualChar + 1)];
```

Comment:
does it work with something like this: `?param1=value1&param2=is=containing=some=equals&param3=value3`

Reply:
I added a more complex test to cover that ;)